model/compile/QtiAssetCompiler/QtiItemAssetCompiler.php

```diff
@@ -50,9 +51,7 @@ public function extractAndCopyAssetFiles(Item $qtiItem, Directory $publicDirecto
         $replacementList = $packedAssets = [];
```

Comment:
This is no longer needed. Also the calls must be fixed: https://github.com/oat-sa/extension-tao-itemqti/pull/1517/files#diff-e8652df0cba6546296d14104113388f6R61